airqo-platform/AirQo-frontend
# [NetManager] Components Lazy loading

#### Summary of Changes (What does this PR do?)
- Implement component lazy loading to improve to page load time

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Fetch the branch locally
* Install dependencies `npm install`
* Start the application `npm run stage-mac` (macOS) or `npm run stage-pc` (Windows platform)
##### Testing on Chrome Browser
* Load the application on Chrome
* Open `Chrome developer console`. 
* Navigate to the network panel and check the disable cache option. To view the changes, may need to to use `slow3G` option.
* Reload the app with the console still open.
* Navigate any any other page using the sidebar tab, you should see the fallback loader while the component resources are being fetched from the server.

#### What are the relevant tickets?
- [PLAT-157](https://airqoteam.atlassian.net/browse/PLAT-157)

#### Screenshots (optional)
